reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-71.md

```diff
@@ -0,0 +1,652 @@
+- Don't write DNS resolution errors on Test-Connection -Quiet (#12204) (Thanks @vexx32!)
+- Use dedicated threads to read the redirected output and error streams from the child process for out-of-proc jobs (#11713)
+- **v7.1.0-preview.1**
+- Fix an operator preference order issue in binder code (#12075) (Thanks @DamirAinullin!)
+- Fix NullReferenceException when binding common parameters of type ActionPreference (#12124)
+- Fix default formatting for deserialized MatchInfo (#11728) (Thanks @iSazonov!)
+- Use asynchronous streams in Invoke-RestMethod (#11095) (Thanks @iSazonov!)
+- Address UTF-8 Detection In Get-Content -Tail (#11899) (Thanks @NoMoreFood!)
+- Handle the IOException in Get-FileHash (#11944) (Thanks @iSazonov!)
+- Change 'PowerShell Core' to 'PowerShell' in a resource string (#11928) (Thanks @alexandair!)
+- Bring back MainWindowTitle in PSHostProcessInfo (#11885) (Thanks @iSazonov!)
+- Miscellaneous minor updates to Windows Compatibility (#11980)
+- Fix ConciseView to split PositionMessage using [Environment]::NewLine (#12010)
+- Remove network hop restriction for interactive sessions (#11920)
+- Fix NullReferenceException in SuspendStoppingPipeline() and RestoreStoppingPipeline() (#11870) (Thanks @iSazonov!)
+- Generate GUID for FormatViewDefinition InstanceId if not provided (#11896)
+- Fix ConciseView where error message is wider than window width and doesn't have whitespace (#11880)
+- Allow cross-platform CAPI-compatible remote key exchange (#11185) (Thanks @silijon!)
+- Fix error message (#11862) (Thanks @NextTurn!)
+- Fix ConciseView to handle case where there isn't a console to obtain the width (#11784)
+- Update CmsCommands to use Store vs certificate provider (#11643) (Thanks @mikeTWC1984!)
+- Enable pwsh to work on Windows systems where mpr.dll and STA is not available (#11748)
+- Refactor and implement Restart-Computer for Un*x and macOS (#11319)
+- Add an implementation of Stop-Computer for Linux and macOS (#11151)
+- Fix help function to check if less is available before using (#11737)
+- Update PSPath in certificate_format_ps1.xml (#11603) (Thanks @xtqqczze!)
+- Change regular expression to match relation-types without quotes in Link header (#11711) (Thanks @Marusyk!)
+- Fix error message during symbolic link deletion (#11331)
+- Add custom 'Selected.*' type to PSCustomObject in Select-Object only once (#11548) (Thanks @iSazonov!)
+- Add -AsUTC to the Get-Date cmdlet (#11611)
+- Fix grouping behavior with Boolean values in Format-Hex (#11587) (Thanks @vexx32!)
+- Make Test-Connection always use the default synchronization context for sending ping requests (#11517)
+- Correct startup error messages (#11473) (Thanks @iSazonov!)
+- Ignore headers with null values in web cmdlets (#11424) (Thanks @iSazonov!)
+- Re-add check for Invoke-Command job dispose. (#11388)
+- Revert "Update formatter to not write newlines if content is empty (#11193)" (#11342) (Thanks @iSazonov!)
```

Comment:
Remove this - since the feature was reverted it does not apply to GA. Also, remove the related item that added the feature being reverted.